docs/documentation.js

```diff
@@ -0,0 +1,69 @@
+const dictum = require('dictum.js');
```

Comment:
you don't need to do all this, you simply need to call dictum inside the tests with the response.